lib/plugins/aws/package/compile/events/cognitoUserPool/index.js

```diff
@@ -103,12 +103,16 @@ class AwsCompileCognitoUserPoolEvents {
 
       const userPoolLogicalId = this.provider.naming.getCognitoUserPoolLogicalId(poolName);
 
+      const DependsOn = _.map(currentPoolTriggerFunctions, (value) => this
+        .provider.naming.getLambdaLogicalId(value.functionName));
+
       const userPoolTemplate = {
         Type: 'AWS::Cognito::UserPool',
```

Comment:
Thanks for the feedback @HyperBrain 👍 

I'm not too familiar with Cognito User Pools. Should we keep it here or remove it? I'm always in favor of having a default (even if it's super rudimentary).